### PR TITLE
MTV 2.5.7 Release notes

### PR DESCRIPTION
**Pull request by RichardHoch**
MTV 2.5.7

Resolves https://issues.redhat.com/browse/MTV-1113 by creating Release notes for MTV 2.5.7

Preview:

[only changes are the links to all known and resolved issues]

### CODE REVIEW

**Comment by RichardHoch** [comment]
@ahadas @anarnold97 Please review.

**Comment by anarnold97** [comment]
@RichardHoch - this is cool, just what i need. 

Just give me a shout when you want me to move it to GitLab

**Review by ahadas — APPROVED** [review]

**Review by ahadas — COMMENTED** [review]
@RichardHoch @anarnold97 I played a bit with the query in Jira to see if we can set `fixVersion < 2.6.0` instead of `fixVersion <= <latest z-stream` and surprisingly I got different results. this test revealed an issue - the versions are compared by the ordered they appear in Jira. up until now I ordered the versions by their release dates - and that's why I got more results for `fixVersion <= 2.5.7` compared to `fixVersion < 2.6.0` (2.5.7 was placed after 2.6.0 and after 2.6.1)

so long story short:
1. I sorted [version in Jira](https://issues.redhat.com/plugins/servlet/project-config/MTV/versions?status=no-filter) by their name
2. now we can have for version 2.X a query that checks `(fixVersion < 2.<X+1>.0 and fixVersion >= 2.X.0)` (for this to work we need to make sure 2.<X+1>.0 exists in Jira when we release 2.X.0 but that shouldn't be an issue)

**Review by ahadas — COMMENTED** [review]
the changes look good, we indeed only need to update these filters for 2.5.7 but how about changing the filter in a similar way to what we did for 2.6.2 so that the filters will be set with `fixVersion < 2.[x+1].0` rather than `fixVersion <= 2.[x].[y]`  (in our case `fixVersion < 2.6.0` instead of `fixVersion <= 2.5.7` )

**Comment by anarnold97** [comment]
@ahadas - i built this to stage last night. I have changed the filters, but will create new filters and change the name of these filters to 2.5.z (Known or Resolved issues) in the 2.5.8 release 

Is this ok with you?

**Comment by ahadas** [comment]
> @ahadas - i built this to stage last night. I have changed the filters, but will create new filters and change the name of these filters to 2.5.z (Known or Resolved issues) in the 2.5.8 release
> 
> Is this ok with you?

yep, that's ok, thanks!

**Review by ahadas — APPROVED** [review]

**Comment by RichardHoch** [comment]
@anarnold97 I've moved the Jira status to Testing. May i move it to "Release Pending" or would you like to do that when you move this to GitLab. Please and thanks for your help with the filters.

**Comment by anarnold97** [comment]
@RichardHoch - i moved this to GitLab last night.

You are good to move to `Release Pending`.

However, please do **NOT MERGE** as I will need this ticket to be `open` when I do the reverse sync from GitLab. 

Thanks
